R/inboveg_recordings.R

```diff
@@ -28,6 +28,7 @@
 #' @importFrom DBI dbGetQuery
 #' @importFrom assertthat assert_that
 #' @importFrom dplyr collect tbl sql
+#' @importFrom inborutils connect_inbo_dbase
```

Comment:
Deze lijn is niet nodig, omdat je al in het inborutils package zit, kent de functie `connect_inbo_dbase()` al.